uint8 action
int32 target
---
bool success